Cabal-hooks/changelog.md

```diff
@@ -1,6 +1,6 @@
 # Changelog for `Cabal-hooks`
 
-## 0.1 – December 2023
+## 3.14 – December 2023

```

Comment:
It's a little suspicious to have 3.14 marked as appearing on Dec 2023, which is long before Cabal 3.14...